src/engines/html5/html5.js

```diff
@@ -880,5 +892,32 @@ export default class Html5 extends FakeEventTarget implements IEngine {
     }
     return null;
   }
+
+  /**
+   * Register the metadata track event
+   * @returns {void}
+   * @private
+   */
+  _registerMetadata(): void {
+    const tracks = this._el.textTracks;
+    for (let track in tracks) {
+      if (tracks.hasOwnProperty(track)){
+        if (tracks[track].kind == "metadata") {
+          tracks[track].mode = "showing";
+          this._registeredMetadataEvent = true;
+          tracks[track].addEventListener("cuechange", (evt) => {
```

Comment:
@itaykinnrot we are already listen to this event and running this logic in the engine, I'm wondering if we can reuse the existing listener to your needs?

Reply:
No since the other event is for the real text - this one is for metadata -- id3 tags